### DISCUSSION TITLE
Looking for guidance

### DISCUSSION THREAD

**Discussion by rsquaredg** [t0sche/cvss-bt#22] [General]
Hello t0sche, came across this while searching for tooling to enrich CVSS data.  The current system is broke and does not lead to actionable data for many vulnerabilities.  Your project seems to solve for some of that.  I am not sure how to use your project.  It looks like cvss-bt is the enriched data set.  Is that the case or is there a different way to use your project?

**Comment by t0sche** [comment]
The project is a self-contained data set that continually updates and publishes new daily CVSS-BT scores enriched with vulnerability intelligence sources.

If your organization relies on CVSS to prioritize vulnerabilities, this would be useful.
